pyfar/classes/audio.py

```diff
@@ -205,14 +206,22 @@ def __init__(self, data, times, comment=""):
         times : array, double
             Times in seconds at which the data is sampled. The number of times
             must match the `size` of the last dimension of `data`.
-        comment : str
-            A comment related to `data`. The default is ``''``, which
+        comment : str, optional
+            A comment related to `data`. The default is ``""``, which
             initializes an empty string.
+        complex : bool, optional
+            A flag which indicates if the raw data are real or complex-valued.
```

Comment:
Explicitly refering to the time data might be more clear
```suggestion
            A flag which indicates if the time data are real or complex-valued.
```